models/engine/db_storage.py

```diff
@@ -13,7 +13,7 @@
 from models.user import User
 from os import getenv
 import sqlalchemy
-from sqlalchemy import create_engine
+from sqlalchemy import create_engine, select
```

Comment:
This line is concise and is doing exactly what it is supposed to. (for creating a connection to a database,  for constructing SQL SELECT queries)